charts/atlas-operator/templates/roles.yaml

```diff
@@ -14,7 +14,9 @@ metadata:
   labels:
   {{- include "mongodb-atlas-operator.labels" $ | nindent 4 }}
 rules:
-{{ template "mongodb-atlas-operator.rbacRules" $ | toYaml | indent 2 }}
+{{- range (.Files.Lines "rbac.yaml") }}
```

Comment:
Why do we need to have the RBAC definitions repeated 3 times?

Reply:
The reason these included twice, is that there are two sections for Roles: one for when AKO works one of managed namespaces, and when AKO is not in the managed namespaces list.

Reply:
It is included 3 times total. One in cluster roles and two here.

I see the ifs now for watched namespaces and not watched. Thanks for explaining.